src/merge.js

```diff
@@ -19,6 +19,9 @@ var _extend = require('./internal/_extend');
  *      R.merge({ 'name': 'fred', 'age': 10 }, { 'age': 40 });
  *      //=> { 'name': 'fred', 'age': 40 }
  *
+ *      R.merge({ 'name': 'fred', 'age': 10 }, { 'age': undefined });
+ *      //=> { 'name': 'fred', 'age': undefined }
```

Comment:
Adding an `@see`  link to `assoc` and especially `dissoc` does sound appropriate, though.